device-type/overlay-car/frameworks/base/core/res/res/values/config.xml

```diff
@@ -5,4 +5,5 @@
     <bool name="config_supportsSplitScreenMultiWindow">false</bool>
     <!-- Enable AppWidgetService even the FEATURE_APP_WIDGETS is disable -->
     <bool name="config_enableAppWidgetService">true</bool>
+     <bool name="config_voice_capable">true</bool>
```

Comment:
If I remember correctly, we decided to not set this and instead have a google diff patch in dialer to make sure voice calls are allowed irrespective of this setting